context/package.json

```diff
@@ -82,5 +84,13 @@
     "test:coverage": "jest --coverage",
     "lint": "eslint -c .eslintrc.yml --ext .js,.jsx app/static/js/",
     "lint:fix": "eslint -c .eslintrc.yml --ext .js,.jsx --fix app/static/js/"
+  },
+  "lint-staged": {
+    "app/static/js/**/*.{js,jsx}": [  "prettier --write", "eslint -c .eslintrc.yml"]
```

Comment:
Would the changes from `prettier` be a subset of those from `eslint --fix`? Why not the latter?

Reply:
If you're comfortable with `eslint --fix` that would be the only command needed.

Reply:
@mccalluc I'm going to merge, but let me know if you're comfortable using `eslint --fix` and I'll make another PR.